study-builder/studies/testboston/emails/account_created.html

```diff
@@ -1,28 +1,275 @@
 </head>
 <body>
-<p>-ddp.salutation-</p>
+<center class="wrapper" data-link-color="#2660B2" data-body-style="font-size:16px; font-family:arial,helvetica,sans-serif; color:#464f61; background-color:#ffffff;">
+  <div class="webkit">
+    <table cellpadding="0" cellspacing="0" border="0" width="100%" class="wrapper" bgcolor="#ffffff">
+      <tr>
+        <td valign="top" bgcolor="#ffffff" width="100%">
+          <table width="100%" role="content-container" class="outer" align="center" cellpadding="0" cellspacing="0" border="0">
+            <tr>
+              <td width="100%">
+                <table width="100%" cellpadding="0" cellspacing="0" border="0">
+                  <tr>
+                    <td>
+                      <!--[if mso]>
+                      <center>
+                        <table><tr><td width="600">
+                      <![endif]-->
+                      <table width="100%" cellpadding="0" cellspacing="0" border="0" style="width:100%; max-width:600px;" align="center">
+                        <tr>
+                          <td role="modules-container" style="padding:0px 0px 0px 0px; color:#464f61; text-align:left;" bgcolor="#ffffff" width="100%" align="left"><table class="module preheader preheader-hide" role="module" data-type="preheader" border="0" cellpadding="0" cellspacing="0" width="100%" style="display: none !important; mso-hide: all; visibility: hidden; opacity: 0; color: transparent; height: 0; width: 0;">
+                            <tr>
+                              <td role="module-content">
+                                <p>You have successfully created a study account at TestBoston.org.</p>
+                                <p>
+                                  However, you are not enrolled in the study until you read the Informed Consent.
+                                  You must read the Informed Consent study document, and if you wish to enroll,
+                                  provide consent to participate by typing your name into the form.
+                                </p>
+                              </td>
+                            </tr>
+                          </table><table class="wrapper" role="module" data-type="image" border="0" cellpadding="0" cellspacing="0" width="100%" style="table-layout: fixed;" data-muid="4UqFsRLozLcypAAv4CeoFS">
+                            <tbody><tr>
+                              <td style="font-size:6px; line-height:10px; padding:30px 0px 10px 18px;" valign="top" align="left">
+                                <img class="max-width" border="0" style="display:block; color:#000000; text-decoration:none; font-family:Helvetica, arial, sans-serif; font-size:16px; max-width:50% !important; width:50%; height:auto !important;" src="http://cdn.mcauto-images-production.sendgrid.net/39aee68f3d52d7ef/a7a572ef-afb4-4747-a7eb-025d3139ab3e/1791x246.png" alt="TestBoston" width="300" data-responsive="true" data-proportionally-constrained="false">
```

Comment:
Can we change the image source to something that we control instead of `http://cdn.mcauto-images-production.sendgrid.net/...`? Maybe we need a bucket? Or just reference `{{baseWebUrl}}/assets/images/logo-testboston.png` (like https://testboston.dev.datadonationplatform.org/assets/images/logo-testboston.png)?

Reply:
Or at the very least, we should make sure this image URL is stable enough.

Reply:
Having a non substitution image url is nice because you can see the image in preview. 
Any way changed it to {{baseWebUrl}}/assets/images/logo-testboston.png